### PR TITLE
Increase force and throwforce of intact bottles, bottles guaranteed to break on impact

### PR DESCRIPTION
**Pull request by ErisMeleeEnjoyer**
## About The Pull Request

force from 5 to 7
throwforce from 1(__!!!__) to 7
tweaked intent check on throwing bottles
## Why It's Good For The Game
first off, now molotovs ACTUALLY WORK
second, hitting someone with a bottle is far stronger than a *fucking punch* now
third, throwing a bottle at someone is now ~~an alternative to throwing knives~~ *actually useful*
## Changelog
:cl:
add: increase force and throwforce of intact bottles
tweak: bottles guaranteed to break on impact when not in help mode
/:cl:

### CODE REVIEW

**Comment by Alliostra** [comment]
I think making bottles not break when thrown on help intent is good, handling bottles especially as club personnel would be much easier this way.

**Comment by ErisMeleeEnjoyer** [comment]
people can catch bottles if they are on active throw intent when the bottle collides with them. I see this as an issue of skill

**Comment by Rexdyer1** [comment]
> people can catch bottles if they are on active throw intent when the bottle collides with them. I see this as an issue of skill

It does add extra danger when throwing bottles into the trash. the help intent could be useful.

**Comment by TheShown911** [comment]
I support the help intent idea.

**Comment by Pink-Chink** [comment]
Damage seems a bit too high compared to throwing knives, which are an actual weapon.

**Comment by ErisMeleeEnjoyer** [comment]
also they, like, break after impact and don't embed. pure concussion force.

**Comment by Pink-Chink** [comment]
I'd say throwforce = WEAPON_FORCE_WEAK and force = WEAPON_FORCE_WEAK. Still a buff and can be adjusted later. I don't think they should be better than the knives in any way. Except for the niche use of applying chems.

File: code/modules/reagents/reagent_containers/food/drinks/bottle.dm [hunk 1]
```diff
@@ -5,8 +5,10 @@
 /obj/item/reagent_containers/food/drinks/bottle
 	amount_per_transfer_from_this = 10
 	volume = 100
+	description_info = "Thrown bottles don't break when you throw them while being on help intent."
 	item_state = "broken_beer" //Generic held-item sprite until unique ones are made.
-	force = 5
+	force = 20
+	throwforce = 20
```
Comment (Pink-Chink) [inline code/modules/reagents/reagent_containers/food/drinks/bottle.dm:11]:
```suggestion
	force = WEAPON_FORCE_WEAK
	throwforce = WEAPON_FORCE_WEAK
```

**Comment by ErisMeleeEnjoyer** [comment]
oh god that's so low aaa

**Review by Pink-Chink — APPROVED** [review]

**Pull request merged**